### ISSUE TITLE
PDF Upload issue

### ISSUE THREAD

**Issue by trchiao** [the-core-studio/vox-issue#223]
- Is it supported across the platform
- Size Limit?

**Comment by tclzcja** [comment]
It seems like, there IS a size limit on PDF uploading, only that the limit is setup by the ITS, not us.

I've tried to upload a 57kb PDF and it works perfectly, yet another 200kb PDF always failed.

**Comment by tclzcja** [comment]
Working with ITS to solve this. We can't tolerate this anymore.

**Comment by tclzcja** [comment]
![Image](https://github.com/user-attachments/assets/5956fa1f-fa69-4d12-bda1-5c18ccfda5fa)

Confirmed with ITS that it's blocked by their application firewall.

**Comment by tclzcja** [comment]
Pasting ITS's résponse here:

Dear JingAn,

In your case we cannot 100% it's a false alarm or not. As some of the PDF files are able to upload.
We must maintain the highest level of protection to the server we support, exception for file upload is not a good way to solve this. 
And like you said only a few users have these issues. 

The system is successfully confirm the upload file type is PDF format, and scan the contents with XSS.
The problem may relate to the original like using outdated software and created pdf files, and Adobe PDF reader can be used to execute malicious script implanted by attacker.

https://owasp.org/www-community/attacks/xss/
https://www.broadcom.com/support/security-center/attacksignatures/detail?asid=22062

**Comment by tclzcja** [comment]
I agree with ITS that the content of the PDF may be vulnerable to XSS attack. This may due to using an old version of Adobe Reader to create/save the PDF file. Consider that this is University, it's possible a lot of education PDF were made years ago, when potential XSS breaches may not be a thing back then, and will be blocked by latest application firewall rules.

**Comment by tclzcja** [comment]
We should modify our file upload code, maybe uploading to TALIC cloud directly in front-end, or try using form multipart upload.

**Comment by tclzcja** [comment]
This is done. I used a PDF "wash" function in the Front End to de-sensitize every PDF upload. Now any PDF can be uploaded, through MediaNode upload and Toolbox assets import.
